frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigForm.js

```diff
@@ -24,36 +24,52 @@ const useStyles = makeStyles({
 export default function ConfigForm({ addConfig }) {
   const classes = useStyles();
 
+  const DEFAULT_USER = "example@gmail.com";
+
   const [config, setConfig] = useState({
-    id: '',
-    data: '',
-    relatedData: '',
+    user: DEFAULT_USER,
+    dimension: '',
+    metric: '',
+    relatedDimension: '',
+    relatedMetric: '',
   });
 
-  const DEFAULT_ID = 10;
-
-  {/**TODO: Replace drop-down menu and hard-coded data options with text-input that recommends data options*/}
-  const POSSIBLE_DIMENSIONS = ["noodle", "spice", "egg", "soup", "instant noodle"];
+  const POSSIBLE_DIMENSIONS = ["Ramen", "Pizza", "Udon", "Rice", "Japan"];
+  const POSSIBLE_METRICS = ["Interest Over Time - US", "Interest Over Time - UK", "Interest Over Time - JP", "Interest Over Time - Web Search", "Interest Over Time - Images", "Interest Over Time - YouTube"];
   const dimensions = POSSIBLE_DIMENSIONS;
   const relatedDimensions = POSSIBLE_DIMENSIONS;
+  const metrics =  POSSIBLE_METRICS;
+  const relatedMetrics =  POSSIBLE_METRICS;
 
-  function handleDataChange(event) {
-    setConfig({ ...config, data: event.target.value });
+  function handleDimensionChange(event) {
+    setConfig({ ...config, dimension: event.target.value });
   }
 
-  function handleRelatedDataChange(event) {
-    setConfig({ ...config, relatedData: event.target.value });
+  function handleMetricChange(event) {
+    setConfig({ ...config, metric: event.target.value });
   }
 
+  function handleRelatedDimensionChange(event) {
+    setConfig({ ...config, relatedDimension: event.target.value });
+  }
+
+  function handleRelatedMetricChange(event) {
+    setConfig({ ...config, relatedMetric: event.target.value });
+  }
+  
   function handleSubmit(event) {
+    const DELIMITER = "%"
     event.preventDefault();
-    if (config.data.trim() && config.relatedData.trim()) {
-      addConfig({ ...config, id: DEFAULT_ID });
-      setConfig({ ...config, data: "", relatedData: ""});
+    if (config.dimension.trim() && config.metric.trim() && config.relatedDimension.trim() && config.relatedMetric.trim()) {
+      fetch("/api/v1/configurations", {
+        method: 'POST',
+        body: config.user + DELIMITER + config.metric + DELIMITER + config.dimension + DELIMITER + config.relatedMetric + DELIMITER + config.relatedDimension,
```

Comment:
don't have to do this for the PR but fyi for your own knoweldge --  you can use the .join method which might be cleaner since you dont have to keep adding the DELIMITED in between
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join